src/tests/APICore.spec.ts

```diff
@@ -20,43 +20,44 @@ describe('APICore', () => {
     describe('calling the right endpoint', () => {
```

Comment:
Many changes in this test file, but all of them are related to the same thing: The previous tests did not await, and the fetch mock would immediately return.

Versus now, with the backoff mechanism, it's always non blocking, even for tests.